code/datums/emergency_calls/cryo_marines.dm

```diff
@@ -95,9 +95,33 @@
 /obj/effect/landmark/ert_spawns/distress_cryo
 	name = "Distress_Cryo"
 
-/datum/emergency_call/cryo_squad/tech
-	name = "Marine Cryo Reinforcements (Tech)"
-	mob_max = 5
-	max_engineers = 1
-	max_medics = 1
-	max_heavies = 0
+/proc/get_marines()
```

Comment:
Highly suggesting to make a proper path for this. Right now anything and anywhere in the codebase can call this, does it need to be like that?  
e.g. 
`/datum/emergency_call/cryo_squad/tech/proc/get_marines()`
Also move it down below the `/datum/emergency_call/cryo_squad/tech/` after the path is given.